src/makeCalculator.js

```diff
@@ -1,10 +1,58 @@
 'use strict';
 
-/**
- * @return {object}
- */
 function makeCalculator() {
-  // write code here
+  return {
+    result: 0,
+
+    operate(callback, number) {
+      this.result = callback(this.result, number);
+
+      return this;
+    },
+
+    add(operand, number) {
+      const result = operand + number;
+
+      return result;
+    },
+
+    subtract(operand, number) {
+      const result = operand - number;
+
+      return result;
+    },
+
+    multiply(operand, number) {
+      const result = operand * number;
+
+      return result;
+    },
+
+    divide(operand, number) {
+      const result = operand / number;
+
+      return result;
```

Comment:
The `divide` method should use `this.result` instead of `operand`. Additionally, it should handle division by zero to prevent errors. Consider adding a check to ensure `number` is not zero before performing the division.